lib/contentful/client.rb

```diff
@@ -296,6 +298,16 @@ def sync(options = { initial: true })
 
     private
 
+    def ensure_select_is_doable(query)
+      return unless query.key?(:select)
+
+      query[:select] = query[:select].split(',').map(&:strip) if query[:select].is_a? String
```

Comment:
Sending a String is acceptable, as the end result should be the same, but I need to operate with whatever is there, so coercing it to an array is much simpler than having 2 different cases for doing the same.